modules/search/customize-controls/class-label-control.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * A label-only Customizer control for use with Jetpack Search configuration
+ *
+ * @package jetpack
+ * @since 8.6.0
+ */
+
+/**
+ * Label Control class.
+ */
+class Label_Control extends WP_Customize_Control {
+	/**
+	 * Enqueue styles related to this control.
+	 */
+	public function enqueue() {
+		require_once dirname( dirname( __FILE__ ) ) . '/class.jetpack-search-helpers.php';
```

Comment:
You can use `dirname( __FILE__, 2 )` to avoid repeating dirname 👍

Reply:
According to PHPCS, `dirname()` doesn't have a "levels" parameter in PHP version 5.6 or earlier. Since Jetpack supports PHP 5.6, I'll keep this as-is.